src/main/java/com/archisacademy/ecommerceoctobersubs1/model/Wishlist.java

```diff
@@ -0,0 +1,22 @@
+package com.archisacademy.ecommerceoctobersubs1.model;
+
+import jakarta.persistence.*;
+import lombok.Data;
+
+import java.util.List;
+import java.util.UUID;
+
+@Entity
+@Table(name = "wishlists")
+@Data
+public class Wishlist {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "id")
+    private Integer id;
+    @Column(name = "uuid")
+    private String UUID;
+    @Column(name = "product_uuids")
+    private List<String> productUUIDs;
```

Comment:
It is a bit challenging storing in the format of List into Data base. Instead we can do a trick here.
`private String productUUIDs;` while add the uuid we will separate them using `,` and will add them into a list. So please update it as `private String productUUIDs`